x11-terms/terminator/files/terminator-1.91-without-icon-cache.patch

```diff
@@ -0,0 +1,11 @@
+--- a/setup.py.orig	2009-08-12 22:22:53.000000000 -0400
```

Comment:
Short description at the top please.

Reply:
I don't know what this does. This is copied verbatim from 0.90-without-icon-cache.patch but fixed to be compatible with EAPI 7.

Reply:
In the build script, it does this by default:

```python
subprocess.call(["gtk-update-icon-cache", "-q", "-f", "-t", os.path.join(self.install_dir, "share/icons/hicolor")])
```

Turning this to True skips that call, so `icon-theme.cache` is not created. Perhaps it was turned off because terminator shouldn't touch /usr/share/icons/hicolor which is not limited to terminator?

I see /usr/share/icons/hicolor/icon-theme.cache on my machine but it doesn't belong to any package.

Reply:
...but it does get regenerated when I re-emerge terminator regardless of the patch. Because `xdg_pkg_postinst` calls `gtk-update-icon-cache` itself.

Reply:
Updated the description of the patch.

Reply:
thanks for doing it. btw instead of duplicating the patch you could have easily changed the old patch to comply with EAPI-7, as long as making sure it still applies to the old ebuild (most of the time they do).